src/TBuffer.cpp

```diff
@@ -613,6 +613,11 @@ const QMap<QString, QPair<QString, QVector<QChar>>> TBuffer::csmEncodingTable =
 };
 // clang-format on
 
+// a map of supported MXP elements and attributes
+const QMap<QString, QVector<QString>> TBuffer::mSupportedMxpElements = {
+    {QStringLiteral("send"), {"href", "hint", "prompt"}}
```

Comment:
The send support seems a bit odd as we seem to support it only if it's defined as an element with `!EL`. Raw SEND tags seem unsupported.

Reply:
What makes you think so? I see that that SEND is [pre-populated](https://github.com/Mudlet/Mudlet/blob/development/src/TBuffer.cpp#L784) somewhere